lib/active_stash/version.rb

```diff
@@ -1,3 +0,0 @@
-module ActiveStash
-  VERSION = "0.5.0"
```

Comment:
Removing this could, in theory at least, cause problems for someone if they're referring to it.  You can get GVB to set a `VERSION` constant if you make `git-version-bump` a runtime dependency, `require` it, and then call `VERSION = GVB.version`.  That would then remove the need to change the corresponding spec.

Reply:
ah thanks for picking up on that. I'll fix that up.